wikidict/find_templates.py

```diff
@@ -20,6 +20,9 @@ def find_templates(in_words: Dict[str, str], locale: str) -> None:
     templates: Dict[str, str] = {}
     locale_sections = sections[locale]
     for in_word, code in in_words.items():
+        if locale == "da":
```

Comment:
By the way, I'm not sure if this is really kosher, but `--find-templates` needed this to be able to correctly parse sections.